src/code/UserRuleInstructions.cpp

```diff
@@ -65,8 +65,13 @@ UserRuleInstructions::Evaluate(
 
 	// set the named parameters
 	StringList::Iterator it = fParameterNames.GetIterator();
-	for (size_t i = 0; i < parameterCount && it.HasNext(); i++)
-		localVariables.Set(it.Next(), parameters[i]);
+	for (size_t i = 0; it.HasNext(); i++) {
+		if (i < parameterCount)
+			localVariables.Set(it.Next(), parameters[i]);
+		else
```

Comment:
I would use curly braces for the if and else blocks, especially since the else block has two lines.

Reply:
Good point; done.